docs/src/pages/components/tabs/demo.tsx

```diff
@@ -1,23 +1,48 @@
 import * as React from 'react';
 import { Tabs, TabItem, Flex, Button, View } from '@aws-amplify/ui-react';
 
-import { TabsPropControls } from '@/components/TabsPropControls';
-import { useTabsProps } from '@/components/useTabsProps';
-import { Example } from '@/components/Example';
+import { Demo } from '@/components/Demo';
+import { TabsPropControls } from './TabsPropControls';
+import { useTabsProps } from './useTabsProps';
+
+const propsToCode = (props) => {
+  return (
+    `<Tabs` +
+    `\n  currentIndex={${props.currentIndex}}` +
+    `${props.spacing ? `\n  spacing={${props.spacing}}` : ``}` +
+    `\n  justifyContent={${props.justifyContent}}` +
+    `${
+      props.indicatorPosition
+        ? `\n  indicatorPosition={${props.indicatorPosition}}`
+        : ``
+    }` +
+    `>
+  <TabItem title="Tab 1">
+    Tab content #1
+  </TabItem>
+  <TabItem title="Tab 2">
+    Tab content #2
+  </TabItem>,
+  <TabItem title="Disabled" isDisabled={true}>
+    Cannot click
+  </TabItem>
+</Tabs>`
+  );
+};
 
 export const DemoTabPanel = ({ children }) => {
   return <View padding="var(--amplify-space-medium">{children}</View>;
 };
 
 const demoChildren = [
   <TabItem key={1} title="Tab 1">
-    <DemoTabPanel>Tab content #1</DemoTabPanel>
+    Tab content #1
```

Comment:
I removed these `<DemoTabPanel>` from the Demo because it seems nice to have an example of the component out of the box, and with shorter example code (though maybe doesn't look as great without the padding now).

Reply:
Yeah good idea. Should we delete lines 8-10 then?

Reply:
It is used in a lot of the other examples further down on the page so I left it in 😬 Maybe something to clean up for one of the next updates listed? "Move examples to examples directory and use and components"